### PR TITLE
Update/l8 9 naming

### PR DESCRIPTION
**Pull request by dthiex**
Internal issue: https://git.sinergise.com/team-6/openeo-platform/-/issues/83

I am not sure if renaming the yaml files is a good idea as it will break existing links people have to the public collections repository (e.g. https://collections.eurodatacube.com/landsat-8-l1/). Don't think this is really used by anyone anywhere so probably it would be okay to updated.

### CODE REVIEW

File: collections/landsat-8-9-l2.yaml [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+  [Landsat 8-9 Level 2 data](https://www.usgs.gov/core-science-systems/nli/landsat/landsat-collection-2-level-2-science-products)
   from the most recently released [collection 2](https://www.usgs.gov/faqs/what-updates-are-being-made-landsat-collection-2?qt-news_science_products=0#qt-news_science_products), 
   provides atmospherically corrected Surface Reflectance and Surface Brightness Temperature products generated from Collection 2 Level-1 scenes that have been processed to Tier 1 or Tier 2.
-  Collection 2 level 2 data are available since February 2013 and new data are continously added usually within 24 hours after Level 1 scenes are available.
+  Collection 2 level 2 data are available since February 2013 for Landsat 8 and since January 2022 for Landsat 9 and new data are continously added usually within 24 hours after Level 1 scenes are available.
 Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/landsat-8-l2/)"
 Image: landsat-8-l2/landsat-8-l2.png 
 EOBrowser: https:sentinelshare.page.link/X3Hc
-Resolution: 30m
+Resolution: 30 m
```
Comment (dgostencnik) [inline collections/landsat-8-9-l2.yaml:15]: Not important, but other collections don't have space between number and unit

**Comment by dgostencnik** [comment]
I also think filenames shouldn't be changed as someone might be using it

File: package-lock.json [hunk 2]
```diff
@@ -811,6 +811,7 @@
         "jest-resolve": "^26.6.2",
         "jest-util": "^26.6.2",
         "jest-worker": "^26.6.2",
+        "node-notifier": "^8.0.0",
```
Comment (dgostencnik) [inline package-lock.json:814]: this file was probably commited by accident as ?

**Review by dgostencnik — APPROVED** [review]
fftr and merge if you think it is ok to change filenames

File: collections/landsat-8-9-l2.yaml (hunk 1 above)
Comment (dthiex) [reply collections/landsat-8-9-l2.yaml]: Removed the space again for consistence but believe it's more correct to always use one there. Not super important though.

File: package-lock.json (hunk 2 above)
Comment (dthiex) [reply package-lock.json]: Yes, somehow missed that this got updated. Removed the changed to the file when reverting to keep the current file names.

**Comment by dthiex** [comment]
@dgostencnik Can you please approve again. After I did the changes it's necessary again.

> I also think filenames shouldn't be changed as someone might be using it

I left the current filenames as you are right and someone might using it. Breaking this is not worth having slightly more correct file-names.

**Review by dgostencnik — APPROVED** [review]

**Pull request merged**